CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
Just FYI: I have a very busy week (4.5h of training every day from Monday to Thursday) and might not be able to participate in discussions/this PR Review this week. If you explicitly require my input for something, please ping me on Teams. I will only be able to review here again on Friday or next week.

Reply:
Busy week over, review starting now.